hack/common.sh

```diff
@@ -128,9 +132,18 @@ os::build::build_binaries() {
     for platform in "${platforms[@]}"; do
       os::build::set_platform_envs "${platform}"
       echo "++ Building go targets for ${platform}:" "${targets[@]}"
-      go install "${goflags[@]:+${goflags[@]}}" \
-          -ldflags "${version_ldflags}" \
-          "${binaries[@]}"
+      for b in ${binaries[@]}; do
+          echo "Building $b"
+          if [[ "$b" == *"e2e.test"* ]]; then
```

Comment:
I think the fact that you have a completely separate branch for the e2e.test binary is a good indication that it shouldn't be in the same list as cmd/openshift and cmd/oc